libs/content/workspace-ffi/src/apple/macos/api.rs

```diff
@@ -27,15 +27,15 @@ pub unsafe extern "C" fn key_event(
 
     let Some(key) = NSKeys::from(key_code) else { return };
 
-    // Event::Text
-    let text_modifiers = modifiers.shift_only() || modifiers.is_none();
-    if pressed && text_modifiers && key.valid_text() {
-        let text = CStr::from_ptr(characters).to_str().unwrap().to_string();
-        obj.raw_input.events.push(Event::Text(text));
-    }
+    let alt_without_ctrl_cmd =
+        modifiers.alt && !modifiers.ctrl && !modifiers.mac_cmd && !modifiers.command;
```

Comment:
Could this be `alt_only`?

Reply:
Also you don't need to check `mac_command` (like command but excludes ctrl on windows/linux)

Reply:
You can insert some characters using `alt` + `shift`. I didn't know that, I'll remove regular command in that check then.